libs/permissions/permission-endpoint/src/main/kotlin/net/corda/libs/permissions/endpoints/v1/permission/PermissionEndpoint.kt

```diff
@@ -15,24 +15,52 @@ import net.corda.libs.permissions.endpoints.v1.permission.types.PermissionRespon
  */
 @HttpRpcResource(
     name = "RBAC Permission API",
-    description = "Role Based Access Control Permissions Management endpoints.",
+    description = "The RBAC Permission API consists of a number of endpoints enabling permissions management in the " +
+            "RBAC (role-based access control) permission system. You can get details of specified permissions " +
+            "and create new permissions.",
     path = "permission"
 )
 interface PermissionEndpoint : RpcOps {
 
     /**
      * Create a permission entity in the RBAC permission system.
      */
-    @HttpRpcPOST(description = "Create a Permission")
+    @HttpRpcPOST(description = "This method creates a new permission.", responseDescription = """
+        id - Server-side generated ID of the new permission.
+        permissionType - Defines whether this is an ALLOW or DENY type of permission. 
+        permissionString - Machine-parseable string representing an individual permission. 
+            It can be any arbitrary string as long as the authorization code can make use of it in the context of user permission matching.
+        groupVisibility - Optional group visibility identifier of the Permission.
```

Comment:
```suggestion
        groupVisibility: An optional group visibility identifier of the permission
```

As in previous review I would describe what group visibility is.